src/routers/Stripe/common.ts

```diff
@@ -0,0 +1,130 @@
+import assert from 'assert';
+import { ContextType } from '../../context/Context';
+import { Order } from 'src/modules/order/Models/Order';
+import { WhpptMongoDatabase } from 'src/Services/Database/Mongo/Database';
```

Comment:
This should not be needed. We risk opening more connections

Reply:
Will discuss